src/drivers/driver_zephyr.c

```diff
@@ -0,0 +1,540 @@
+/*
+ * Driver interaction with Zephyr WLAN device drivers.
+ * Copyright (c) 2022, Nordic Semiconductor
+ *
+ * This software may be distributed under the terms of the BSD license.
+ * See README for more details.
+ */
+#include "includes.h"
+#include "utils/common.h"
+#include "eloop.h"
+#include "driver_zephyr.h"
+
+K_MBOX_DEFINE(wpa_supp_mbox);
+
+static void wpa_supp_drv_mbox_msg_handler(void *eloop_ctx, void *timeout_ctx)
+{
+	struct k_mbox_msg msg;
+	struct zep_wpa_supp_mbox_msg_data mbox_msg_data;
+
+	/* Prepare to receive message */
+	msg.info = 10000;
+	msg.size = 10000;
```

Comment:
Why are those hardcoded to such value? Should this rather reflect the expected message size?